src/assets/scss/_science.scss

```diff
@@ -26,6 +26,10 @@
     display: none;
 }
 
+.figure-img {
```

Comment:
I wasn't sure about the side-effects of adding this. It's probably OK though.

Reply:
I wasn't able to see any side effects when looking over the the website and also couldn't find instances of this css class, so that should be fine